<?xml version="1.0"?>
<launch>
	<group ns="intaking">
		<rosparam file="$(find behaviors)/config/2023_intaking_config.yaml" command="load"/>
		<node name="intaking_server_2023" pkg="behaviors" type="2023_intaking_server" output="screen"/>
	</group>
</launch>
